oh_queue/static/js/components/ticket_layout.js

```diff
@@ -0,0 +1,63 @@
+class TicketLayout extends React.Component {
+  componentWillMount() {
+    let id = +this.props.match.params.id;
+    let state = this.props.state;
+    let ticket = getTicket(state, id);
+    if (ticket == null) {
+      this.props.loadTicket(id);
+    }
+  }
+
+  render() {
+    let id = +this.props.match.params.id;
+    let state = this.props.state;
+    let ticket = getTicket(state, id);
+
+    if (ticket == null) {
+      if (isLoading(state, id)) {
+        return null;  // TODO loading indicator instead of blank screen
+      } else {
+        return <NotFound/>;
+      }
+    }
+
+    if (!isStaff(state) && !ticketIsMine(state, ticket)) {
+      return <NotFound/>;
+    }
+
+    let assignment = ticketAssignment(state, ticket);
+    let location = ticketLocation(state, ticket);
+    let question = ticketQuestion(state, ticket);
+
+    return (
+      <div>
+        <Navbar state={state}/>
+        <div className="container">
+          <Messages messages={state.messages}/>
+          <div className="row ticket">
+            <div className="col-xs-12">
+              <h2 className="ticket-view-name text-center">
+                { (ticket.status === 'pending' && isStaff(state)) ? 'Help to View Name' : ticket.user.name }
+                <small className="clearfix">{ assignment.name } Q{ question } &middot; { location.name } </small>
```

Comment:
Intentionally prefixes question numbers, like `Q3`

Reply:
Right but this breaks the ticket view page since `ticketQuestion` already prepends Q.

![image](https://user-images.githubusercontent.com/2563849/64307131-00d50200-cf4a-11e9-84e3-12fee81866cb.png)